designs/virtualization/4.x/Scheduling section/Firmware-config/Firmware-config.md

```diff
@@ -0,0 +1,38 @@
+---
+parent: virtualization
+version: 4.5
+---
+
+# Firmware
+
+Exposing Firmware Config for VM.
+Firmware is placed within the scheduling section. It is related to CPU, and shoudn't be seperated from the rest of the CPU features. Because of that, it should be located next to other CPU features. Also, it affects performance, so it is related to "Resource Requirements".
+
+![VM page](img/Firmware-0-0.jpg)
```

Comment:
nit: can we make sure we don't have screens that show VMs VMTs in the side navigation and just have Virtualization. Someone actually pointed it out the other day at a demo and was confused by it :)